estela-api/api/views/auth.py

```diff
@@ -260,8 +293,9 @@ def get_parameters(self, request):
         methods=["POST"],
         detail=False,
         serializer_class=ResetPasswordRequestSerializer,
+        url_path="request",
     )
-    def request(self, request, *args, **kwargs):
+    def request_reset(self, request, *args, **kwargs):
```

Comment:
Tags were not generated properly for this one (defaulting to `api`). Probably because behind of that inheritance there is already a `self.request` used for another thing and `@swagger_auto_schema` was not detecting the tag correctly. So the workaround would be to preserve the same path by using `url_path` and renaming the method.